yamux/src/stream.rs

```diff
@@ -386,35 +398,41 @@ impl io::Read for StreamHandle {
             StreamState::RemoteClosing | StreamState::Closed | StreamState::Reset => (),
             _ => {
                 if self.send_window_update().is_err() {
-                    return Err(io::ErrorKind::BrokenPipe.into());
+                    return Poll::Pending;
```

Comment:
Poll::Pending? IMO, the stream should not be polled again.

Reply:
yeah，you are right